client/verta/docs/tutorials/deployment/endpoint_update.rst

```diff
@@ -0,0 +1,58 @@
+Model Deployment
+================
+
+After an Endpoint has been `created <endpoint_creation.html>`__, it needs to be updated with a
```

Comment:
```suggestion
After an Endpoint has been `created <endpoint_creation.html>`_, it needs to be updated with a
```

I think this is the right syntax.

Reply:
They're both valid. Two underscores tells Sphinx "Don't bother adding `created –> endpoint_creation.html` to a global list of link references because I'm only using this once.", but it's not really required.

Reply:
Oh, interesting